get_hashsets.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+const AppConstants = require("./app-constants").init();
+
+const fs = require("fs");
+const path = require("path");
+const request = require("request");
+
+const pkg = require("./package.json");
+
+const HIBP_AUTH = `Bearer ${AppConstants.HIBP_API_TOKEN}`;
+const USER_AGENT = `${pkg.name}@${pkg.version}`;
+const BREACH_HASHSET_DIR = "breach_hashsets";
+
+
+function getBreachHashset(breach) {
+  if (breach.IsActive && breach.IsVerified && breach.DataClasses.includes("Email addresses")) {
```

Comment:
Not urgent, but a comment documenting the structure of the `breach` object would be nice!